### ISSUE TITLE
I have assets that are not working, i tried multiple solutions with no effect

### ISSUE THREAD

**Issue by MrPaz28** [flutter/flutter#65251]
## Steps to Reproduce

i was following net ninjas tutorial for flutter, and on episode 33 when he adds assets i get this error

[   +7 ms] Error: unable to find directory entry in pubspec.yaml: D:\Android Studio shite\Projects\world_time\assetfolder\

```
```

warning - The asset directory assetfolder/ does not exist - pubspec.yaml:52:7 - asset_directory_does_not_exist

2 issues found. (ran in 2.1s)

```
```

[√] Flutter (Channel stable, v1.17.5, on Microsoft Windows [Version 10.0.18362.1016], locale en-US)
    • Flutter version 1.17.5 at C:\Src\flutter
    • Framework revision 8af6b2f038 (9 weeks ago), 2020-06-30 12:53:55 -0700
    • Engine revision ee76268252
    • Dart version 2.8.4

[!] Android toolchain - develop for Android devices (Android SDK version 30.0.1)
    • Android SDK at C:\Users\altaj  it\AppData\Local\Android\sdk
    • Platform android-30, build-tools 30.0.1
    • Java binary at: D:\Android Studio shite\Files\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor --android-licenses

[√] Android Studio (version 4.0)
    • Android Studio at D:\Android Studio shite\Files
    • Flutter plugin version 48.0.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] Connected device (1 available)
    • sdk gphone x86 • emulator-5554 • android-x86 • Android 11 (API 30) (emulator)

! Doctor found issues in 1 category.

```
```
basically i had 2 assets, i have correct indentation on pubspec.yaml, then i deleted the folder theyre in, fixed my pubspec.yaml, but whenever i try to run i get issues 

many thanks in advance

**Comment by pedromassangocode** [comment]
Hi @MrPaz28 
It seems you are asking for assistance on personal code. Please see https://flutter.dev/community for resources and asking questions like this,
you may also get some help if you post it on Stack Overflow and if you need help with your code, please see https://www.reddit.com/r/flutterhelp/
Closing, as this isn't an issue with Flutter itself. If you disagree, please write in the comments and I will reopen it.
Thank you
